src/uniprotkb/components/protein-data-views/UniProtKBEntryPublications.tsx

```diff
@@ -35,21 +39,14 @@ const UniProtKBEntryPublications: FC<{
   return (
     <>
       {results &&
-        results
-          .map((literatureItem) => ({
-            ...literatureItem,
-            ...formatCitationData(literatureItem.citation),
-          }))
-          .map(({ citation, statistics, pubmedId, journalInfo }) => (
-            <LiteratureCitation
-              title={citation.title}
-              authors={citation.authors}
-              key={`${citation.title}-${citation.citationType}-${citation.journal}`}
-              pubmedId={pubmedId}
-              statistics={statistics}
-              journalInfo={journalInfo}
-            />
-          ))}
+        results?.map((citationData) => (
```

Comment:
Forgot to check that one again, the previous line isn't needed anymore either. You can remove from the spring branch directly now I guess

Reply:
Argh sorry. Doing this now